astropy/units/tests/test_units.py

```diff
@@ -489,6 +489,13 @@ def test_no_as():
     assert hasattr(u, 'attosecond')
 
 
+def test_no_duplicates_in_names():
+    # Regression test for #5036
+    assert u.ct.names == ['ct', 'count']
+    assert u.ct.short_names == ['ct', 'count']
+    assert u.ct.long_names == ['count']
+    assert set(u.ph.names) == set(u.ph.short_names) | set(u.ph.long_names)
+
```

Comment:
It should be 2 lines in the main body between classes and functions, and 1 line between class methods. However this error is not enforced (E302), thus it doesn't fail the tests. Anyway there are ~370 other cases of this, and it's on my list for the next release to fix these and other PEP8s.